src/netmanager-app/core/apis/MapData.ts

```
@@ -0,0 +1,125 @@
+      
+      export const UserGroups = async (token: string) =>{
+        try {
+                if(token){
+                        const response = await axios.get(`${AirQuoData}?token=${token}`);
+                const data = response.data;
+                return data;
+                }
+                {
+                        console.log("Error Token Is Missing")
+                        return null;
+                }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)